helm/cluster-api-provider-cloud-director/values.yaml

```diff
@@ -19,3 +19,5 @@ resources:
     memory: 512Mi
 
 skipRDE: true
+
+priorityClass: giantswarm-critical
```

Comment:
To make it usable from outside Giant Swarm we should set it to an empty string. Add `if` statement if the value is empty in the deployment and set `priorityClass` globally for cluster-api-provider-cloud-director-app in `config` repo.